asm/macros/event.inc

```diff
@@ -2300,9 +2300,24 @@
 	.byte \sourceId
 	.byte \targetId
 	.endm
-    
+
 	@ set the wild double battle flag
 	@ can be used in conjunection with createmon to set up a wild battle with 2 player mons vs. 1 enemy mon
 	.macro setwilddoubleflag
 	callnative ScriptSetDoubleBattleFlag
 	.endm
+
+    @ When OW_USE_FAKE_RTC and OW_FLAG_PAUSE_TIME is assigned, this macro will stop the flow of time.
```

Comment:
you have spaces instead of tabs :)

assembly (ie .s and .inc files) use tabs, c (.c and .h) uses spaces

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/4954/commits/3d958f9a8690b25ddeea4e04f32a89abcc40d951